src/comp/comp.py

```diff
@@ -24,48 +28,49 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [x.name for x in humans if x.name[0] == "D"]
+
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = []
+b = [x.name for x in humans if x.name[-1] == "e"]
 print(b)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with any letter between 'C' and 'G' inclusive.
 print("Starts between C and G, inclusive:")
-c = []
+c = [x.name for x in humans if x.name[0] in ["C", "D", "E", "F", "G"]]
 print(c)
 
 # Write a list comprehension that creates a list of all the ages plus 10.
 print("Ages plus 10:")
-d = []
+d = list(map(lambda x: x.age + 10, humans))
 print(d)
 
 # Write a list comprehension that creates a list of strings which are the name
 # joined to the age with a hyphen, for example "David-31", for all humans.
 print("Name hyphen age:")
-e = []
+e = [f"{x.name}-{x.age}" for x in humans]
 print(e)
 
 # Write a list comprehension that creates a list of tuples containing name and
 # age, for example ("David", 31), for everyone between the ages of 27 and 32,
 # inclusive.
 print("Names and ages between 27 and 32:")
-f = []
+f = [(x.name, x.age) for x in humans if x.age in range(27, 32+1)]
```

Comment:
Could also say 
```suggestion
f = [(x.name, x.age) for x in humans if x.age in range(27, 33)]
```
Same thing either way